qcschema/dev/wavefunction/result_wavefunction.py

```diff
@@ -0,0 +1,68 @@
+"""
+The primary specified return  wavefunction quantities.
+"""
+
+result_wavefunction = {}
```

Comment:
Did you want to add the `shape` and `items` field to the `result_wavefunction` quantities as well?

Reply:
I ended up changing the result quantites to be pointers to other quantities, e.g. `"orbitals_a": "scf_orbitals_a"`. This is in the docs, but not in the autogenerated part (the descriptive paragraphs for each section of the schema are hard-coded for now.)